app/src/containers/CheckoutPage.tsx

```diff
@@ -601,7 +636,12 @@ class CheckoutPage extends React.Component<CheckoutPageProps, CheckoutPageState>
                 </h2>
               </div>
               <div className="modal-body">
-                <PaymentFormMain onCloseModal={this.handleCloseNewPaymentModal} fetchData={this.fetchOrderData} />
+                {
+                  localStorage.getItem(`${Config.cortexApi.scope}_oAuthRole`) === 'PUBLIC' ? (
+                    <PaymentFormMain defaultPostSelection={false} showSaveToProfileOption={false} onCloseModal={this.handleCloseNewPaymentModal} fetchData={this.fetchOrderData} />)
+                    : (
+                      <PaymentFormMain defaultPostSelection={false} showSaveToProfileOption onCloseModal={this.handleCloseNewPaymentModal} fetchData={this.fetchOrderData} />)
```

Comment:
The formatting here looks a bit off. For readability, can you please try formatting?

Reply:
wip